vita3k/modules/CMakeLists.txt

```diff
@@ -210,6 +210,6 @@ set(SOURCE_LIST
 
 add_library(modules STATIC ${SOURCE_LIST})
 target_include_directories(modules PUBLIC include)
-target_link_libraries(modules PRIVATE xxHash::xxhash)
+target_link_libraries(modules PRIVATE xxHash::xxhash package)
```

Comment:
i more see it inside public juse after, private for me is for external submodule

Reply:
I'll leave it as private then.